public/app/features/search/page/components/FolderSection.tsx

```diff
@@ -38,15 +40,27 @@ export const FolderSection: FC<SectionHeaderProps> = ({ section, selectionToggle
     if (!sectionExpanded) {
       return Promise.resolve([] as DashboardSectionItem[]);
     }
-    let query = {
+    let query: SearchQuery = {
       query: '*',
       kind: ['dashboard'],
       location: section.uid,
     };
     if (section.title === 'Starred') {
-      // TODO
+      const stars = await getBackendSrv().get('api/user/stars');
+      if (stars.length > 0) {
+        query = {
+          uid: stars, // array of UIDs
+        };
+      } else {
+        // ??
```

Comment:
What do we do when nothing is starred?

Reply:
🤔 I think we could keep showing the "starred" folder and we can add a message in the frontend "No dashboards are starred.",  something like:
![EmptyStarred](https://user-images.githubusercontent.com/239999/168599268-af792061-b7b2-467c-8664-017e7cff1624.png)

Not sure if we have something for this pattern from UX point of view, what do you think @amy-super